fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/rdf/LdpTriplePreferences.java

```diff
@@ -25,37 +25,48 @@
 public interface LdpTriplePreferences {
 
     /**
-     * @return Whether to return a minimal container.
+     * What the prefer tag choice is.
      */
-    boolean getMinimal();
+    enum preferChoice {
+        INCLUDE("Include"),
+        EXCLUDE("Exclude"),
+        SILENT("Silent");
+
+        private String value;
+
+        preferChoice(final String value) {
+            this.value = value;
+        }
+    }
 
     /**
-     * @return Whether this prefer tag demands membership triples.
+     * @return Whether this preference tag asks to include or omit a minimal container or is silent.
      */
-    boolean prefersMembership();
+    preferChoice preferMinimal();
```

Comment:
The logic of the pre-PR interface is much simpler and readable than the `!preferences.preferX().equals(EXCLUDE)`, etc. Would it be possible to keep the existing interface and bury the new enum-based logic in the `LdpPreferTag` implementation?